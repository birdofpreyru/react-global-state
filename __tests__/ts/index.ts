test('Library exports match expectations', () => {
  const lib = require('../../src');
  expect(lib).toMatchSnapshot();
});
