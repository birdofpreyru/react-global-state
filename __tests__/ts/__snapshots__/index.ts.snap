// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Library exports match expectations 1`] = `
{
  "AsyncDataEnvelope": [Function],
  "GlobalStateProvider": [Function],
  "SsrContext": [Function],
  "getGlobalState": [Function],
  "getSsrContext": [Function],
  "useAsyncCollection": [Function],
  "useAsyncData": [Function],
  "useGlobalState": [Function],
}
`;
